packages/atlas/src/components/_inputs/VideoLanguageCombobox/VideoLanguageCombobox.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { Meta, Story } from '@storybook/react'
+
+import { VideoLanguageCombobox, VideoLanguageComboboxProps } from './VideoLanguageCombobox'
+
+export default {
+  title: 'inputs/VideoLanguageCombobox',
+  component: VideoLanguageCombobox,
+  args: {
+    onSelectedItemChange: () => null,
+  },
+} as Meta<VideoLanguageComboboxProps>
+
+const Template: Story<VideoLanguageComboboxProps> = (args) => <VideoLanguageCombobox {...args} />
```

Comment:
I thought you'd add the variant in the Combobox stories, something like "Combobox with separator", or "Combobox with suggested options". 
IMO It would be more helpful if we have that story in the Combobox component. It would mean, that adding suggested options is actually possible with the Combobox component. 

Also when the value is not available, I don't see information about a not found item
Combobox
![image](https://user-images.githubusercontent.com/51168865/197516429-f5184aa3-c5bc-4ccd-b502-58b59fbc3ce4.png)
VideoLangaugeCombobox
![image](https://user-images.githubusercontent.com/51168865/197516700-9b36fca0-fea6-479a-a909-6c095973a5e8.png)

Not sure if it's a bug or you just need to provide some props to make this work.